Error in `make initialize`

### Describe the problem and steps to reproduce it:
(Please include as many details as possible.)

- run `make initialize`

### What happened?

This error:

```
TypeError: unhashable type: 'StaticCategory'
make: *** [Makefile-docker:86: populate_data] Error 1
make: *** [initialize] Error 2
```

### What did you expect to happen?

No error.

### Anything else we should know?
(Please include a link to the page, screenshots and any relevant files.)

--- a/src/olympia/landfill/generators.py
+++ b/src/olympia/landfill/generators.py
@@ -80,9 +80,9 @@ def generate_addons(num, owner, app_name, addon_type=ADDON_EXTENSION):
         generate_addon_preview(addon)
         generate_translations(addon)
         # Only feature 5 addons per category at max.
-        if featured_categories[category] < 5:
+        if featured_categories[category.id] < 5:
             generate_collection(addon, app)
-            featured_categories[category] += 1
+            featured_categories[category.id] += 1
         generate_ratings(addon, 5)
 
 

